core/worker/src/main/java/com/cognifide/aet/worker/drivers/FirefoxWebDriverFactory.java

```diff
@@ -82,29 +84,33 @@ public WebCommunicationWrapper createWebDriver(ProxyServerWrapper proxyServer)
       DesiredCapabilities capabilities = new DesiredCapabilities();
       capabilities.setCapability(CapabilityType.PROXY, proxy);
       capabilities.setCapability(CapabilityType.ACCEPT_SSL_CERTS, true);
-
       FirefoxProfile fp = getFirefoxProfile();
       fp.setAcceptUntrustedCertificates(true);
       fp.setAssumeUntrustedCertificateIssuer(false);
-      capabilities.setCapability(AetFirefoxDriver.PROFILE, fp);
+      setCommonCapabilities(capabilities, fp);
 
-      return new WebCommunicationWrapperImpl(getFirefoxDriver(fp, capabilities), proxyServer,
+      return new WebCommunicationWrapperImpl(getFirefoxDriver(capabilities), proxyServer,
           requestExecutorFactory
               .createInstance());
     } catch (Exception e) {
       throw new WorkerException(e.getMessage(), e);
     }
   }
 
+  private void setCommonCapabilities(DesiredCapabilities capabilities, FirefoxProfile fp) {
```

Comment:
Could we move this private method down after the two public methods that use it?

Reply:
moved